tests/lib/prepare-restore.sh

```diff
@@ -301,6 +301,21 @@ prepare_project() {
         fi
     fi
 
+    if [[ "$SPREAD_SYSTEM" == debian-9-* ]]; then
+	# Manually install the latest golang from -backports
+        cat best.py <<'EOF'
+import apt
```

Comment:
We could just have this in `tests/lib/best-golang.py`, is there any reason not to?